lib/tasks/notify.rake

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+namespace :notify do
+  desc "Send an email notification"
+  task :send_email, [:email_address] => :environment do |_, args|
+    raise "Missing email address" unless args.email_address
+
+    headers = {
+      to: args.email_address,
+      template_id: ENV["TEMPLATE_ID"] || "759acac6-da53-4a19-b591-b7538c7c39de",
+      body: ENV["BODY"] || "This is a test email notification",
+      subject: ENV["SUBJECT"] || "Test email notification",
+    }
+
+    message = Mail::Message.new(headers)
+    NotifyDeliveryMethod.new.deliver!(message)
```

Comment:
Are we able to use action mailer for this? Just as it stands we won't be using the action mailer configuration that's been wrote.

I figure we can just do this with something like `ActionMailer::Base.mail(headers).deliver_now` - although there could be a fatal flaw in my plan.

Reply:
I'm getting `OpenSSL::SSL::SSLError: hostname "localhost" does not match the server certificate` when trying that 😕

Reply:
The documentation does suggest that a mailer model needs to be created to use Action Mailer https://apidock.com/rails/ActionMailer/Base

Reply:
Hmm do you have a stack trace of that? do you know if it's coming from notify code or not even making it to that? (could be falling back to a different delivery method maybe)